docs/operator-manual/applicationset/Template.md

```diff
@@ -169,6 +169,9 @@ spec:
     {{- end }}
 ```
 
+!!! important
+    `templatePatch` only works when [go templating](../applicationset/GoTemplate.md) is enabled.
+    This means that the `goTemplate` field under `spec` needs to be set to `true` for template patching to work.
```

Comment:
Just add a space between the sections, otherwise LGTM

```suggestion
    This means that the `goTemplate` field under `spec` needs to be set to `true` for template patching to work.

```